_includes/markdown/Tools.md

```diff
@@ -29,3 +29,22 @@ At 10up, we use [Vagrant](https://www.vagrantup.com/) to build and interact with
 <h3 id="command-line">Command Line Tools</h3>
 
 [WP-CLI](http://wp-cli.org) - A command line interface for WordPress. This is an extremely powerful tool that allows us to do imports, exports, run custom scripts, and more via the command line. Often this is the only way we can affect a large database (WordPress.com VIP or WPEngine). This tool is installed by default on VVV and VIP Quickstart.
+
+<h3 id="a11y-testing">Accessibility Testing</h3>
+
+We use a variety of tools to test our sites for accessibility issues. WebAim has some great resources for [how to evaluate sites](http://webaim.org/articles/screenreader_testing/) with a screen reader.
+
+* [Using VoiceOver](http://webaim.org/articles/voiceover/)
+* [Using NVDA](http://webaim.org/articles/nvda/)
+* [Using JAWS](http://webaim.org/articles/jaws/)
+
+We're also a fan of a few browser tools that lend us a hand when it comes to testing areas like color contrast, heading heirarchy, and ARIA application.
```

Comment:
Hmm was thinking that instead of just adding `<abbr>` changes in this PR, we can make this adjustment across the project as a whole in a new PR. Thoughts?